contracts/upgradeable_contracts/RewardableBridge.sol

```diff
@@ -0,0 +1,63 @@
+pragma solidity 0.4.24;
+
+import "./OwnedUpgradeability.sol";
+import "./Ownable.sol";
+
+
+contract RewardableBridge is Ownable, OwnedUpgradeability {
+
+    function setFee(uint256 _fee) external onlyIfOwnerOfProxy {
```

Comment:
please change the modifier to `onlyOwner` since whether the reward is distributed among the validators is the bridge parameter.

Reply:
Fixed https://github.com/poanetwork/poa-bridge-contracts/pull/132/commits/ee666a15f304de8187c7de8e058ce4314bf38932